src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,13 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+  overflow-x: hidden;
+
+  /* does not work on safari */
+  /* scroll-behavior: smooth; */
```

Comment:
You can enable it on the rest of the browsers. It is an incremental addition, adding it will not break your page on safari. It will improve experience for rest of the users.